### PR TITLE
Sourcing and unknown

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/lib/formatDateClaim.js [hunk 1]
```diff
@@ -68,6 +81,15 @@ function parseDate(wikidatatime, languageCode = "en", yearOnly = false) {
   if (yearOnly && precision > 9) {
     precision = 9;
   }
+  let sourcingPrefix = "";
+  let sourcingPostfix = "";
+  if (sourcingCircumstances === "Q5727902") {
+    //circa
+    sourcingPrefix = "~";
+  } else if (sourcingCircumstances === "Q18122778") {
```
Comment (ogroppo) [inline src/lib/formatDateClaim.js:89]: could go in a const

File: src/lib/formatDateClaim.js [hunk 2]
```diff
@@ -68,6 +81,15 @@ function parseDate(wikidatatime, languageCode = "en", yearOnly = false) {
   if (yearOnly && precision > 9) {
     precision = 9;
   }
+  let sourcingPrefix = "";
+  let sourcingPostfix = "";
+  if (sourcingCircumstances === "Q5727902") {
```
Comment (ogroppo) [inline src/lib/formatDateClaim.js:86]: could go in a const

**Review by ogroppo — APPROVED** [review]
Beautiful!

File: src/lib/formatDateClaim.js (hunk 2 above)
Comment (mshd) [reply src/lib/formatDateClaim.js:86]: ok sure

**Pull request merged**